### PR TITLE
Automated cherry pick of #85620: increase pv controller resync period to try to deflake

### PR DESCRIPTION
**Pull request by msau42**
Cherry pick of #85620 on release-1.17.

#85620: increase pv controller resync period to try to deflake

For details on the cherry pick process, see the [cherry pick requests](https://git.k8s.io/community/contributors/devel/sig-release/cherry-picks.md) page.

```release-note
NONE
```

### CODE REVIEW

**Comment by msau42** [comment]
/hold
wait for CI flakes to look better to confirm test fix.

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by alejandrox1** [comment]
flake?
/test pull-kubernetes-e2e-gce

prev failure: https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/pull/85660/pull-kubernetes-e2e-gce/1199504081069019136/

**Review by alejandrox1 — APPROVED** [review]
/kind flake
/important soon

**Comment by liggitt** [comment]
/lgtm
/approve

Test only change that helps resolve the top integration test flake

Hold until master PR merged

**Comment by justaugustus** [comment]
Release Managers approval:
/lgtm
/approve

**Pull request merged**
